js/bytetrade.js

```diff
@@ -233,8 +233,8 @@ module.exports = class bytetrade extends Exchange {
             let quote = this.safeString (market, 'quoteName');
             const baseId = this.safeString (market, 'base');
             const quoteId = this.safeString (market, 'quote');
-            const normalBase = base.split ('@')[0];
-            let normalQuote = quote.split ('@')[0];
+            const normalBase = base.split ('@' + baseId)[0];
```

Comment:
Hey, @leolianger !

If we cut off the `@NNN` portion – that will effectively overwrite some markets, since the symbol has to be unique.

For example, the ByteTrade API returns two markets with different market ids:

```
    {"amount":{"min":0.0001,"max":-1},"price":{"min":1e-8,"max":-1},"cost":{}} | {"amount":4,"price":8} |           |            | 0.0008 | 0.0008 |   2147483727 |       ROCKET/BTT |      ROCKET |      BTT |     80 |       1 |   true |       ROCKET/BTT
    {"amount":{"min":0.0001,"max":-1},"price":{"min":1e-8,"max":-1},"cost":{}} | {"amount":4,"price":8} |           |            | 0.0008 | 0.0008 |   2147483728 |    ROCKET@81/BTT |   ROCKET@81 |      BTT |     81 |       1 |   true |       ROCKET/BTT
```

```
bytetrade.market (ROCKET@81/BTT)
{       limits: { amount: { min: 0.0001, max: -1 },
                   price: { min: 1e-8, max: -1 },
                    cost: { min: undefined, max: undefined } },
     precision: { amount: 4, price: 8 },
     tierBased:    undefined,
    percentage:    undefined,
         taker:    0.0008,
         maker:    0.0008,
            id:   "2147483728",
        symbol:   "ROCKET@81/BTT",
          base:   "ROCKET@81",
         quote:   "BTT",
        baseId:   "81",
       quoteId:   "1",
          info: {       symbol:   "2147483728",
                          name:   "ROCKET@81/BTT",
                          base:   "81",
                         quote:   "1",
                  marketStatus:    0,
                      baseName:   "ROCKET@81",
                     quoteName:   "BTT",
                        active:    true,
                         maker:   "0.0008",
                         taker:   "0.0008",
                     precision: { amount: 4, price: 8, minPrice: 1 },
                        limits: { amount: { min: "0.0001", max: "-1" },
                                   price: { min: "0.00000001", max: "-1" } } },
        active:   "true",
  normalSymbol:   "ROCKET/BTT"                                                  }
mbp:ccxt igorkroitor$ node examples/js/cli bytetrade market ROCKET/BTT
Node.js: v14.9.0
CCXT v1.35.80
bytetrade.market (ROCKET/BTT)
{       limits: { amount: { min: 0.0001, max: -1 },
                   price: { min: 1e-8, max: -1 },
                    cost: { min: undefined, max: undefined } },
     precision: { amount: 4, price: 8 },
     tierBased:    undefined,
    percentage:    undefined,
         taker:    0.0008,
         maker:    0.0008,
            id:   "2147483727",
        symbol:   "ROCKET/BTT",
          base:   "ROCKET",
         quote:   "BTT",
        baseId:   "80",
       quoteId:   "1",
          info: {       symbol:   "2147483727",
                          name:   "ROCKET/BTT",
                          base:   "80",
                         quote:   "1",
                  marketStatus:    0,
                      baseName:   "ROCKET",
                     quoteName:   "BTT",
                        active:    true,
                         maker:   "0.0008",
                         taker:   "0.0008",
                     precision: { amount: 4, price: 8, minPrice: 1 },
                        limits: { amount: { min: "0.0001", max: "-1" },
                                   price: { min: "0.00000001", max: "-1" } } },
        active:   "true",
  normalSymbol:   "ROCKET/BTT"                                                  }
```

↑ If we remove the `@NNN` from there, one symbol will effectively overwrite another symbol due to a symbolic conflict, since the symbol has to be unique for each market.
Thus, if we cut off `@NNN`, only one of the two markets will remain available.

Reply:
The older version also remove the @NNN, the symbol are still different for  ROCKET/BTT and ROCKET@81/BTT, because we only remove the @NNN for normalSymbol submitted to the chain, because the chain only accepts normalSymbol without "@Id". I think the normalSymbol can be duplicated, because it performed well in the older version.

This PR is to solve this problem, in our test chain, a symbol is called "@AAPL/ETH". If we split it with "@", the normalSymbol will be "/ETH", which is wrong.